cdk/multiplayer_test_scaler/constants.py

```diff
@@ -22,8 +22,8 @@
                        '@echo off\n' \
                        'netsh advfirewall firewall add rule name="O3DE_server" dir=in protocol=UDP localport={server_port} action=allow program="C:\o3de\{project_name}.ServerLauncher.exe" enable=yes\n' \
                        'cd C:/o3de\n' \
-                       '{project_name}.ServerLauncher --engine-path=C:\o3de --project-path=C:\o3de --project-cache-path=C:\o3de\Cache ' \
-                       '--regset="/Amazon/AWSCore/AllowAWSMetadataCredentials=true" ' \
+                       'start /b {project_name}.ServerLauncher --engine-path=C:\o3de --project-path=C:\o3de --project-cache-path=C:\o3de\Cache ' \
+                       '--regset="/Amazon/AWSCore/AllowAWSMetadataCredentials=true" --regset="/O3DE/Metrics/Multiplayer/Active=true" ' \
```

Comment:
Should this be conditional on having metrics stack active for project?

Reply:
are you talking about the `/O3DE/Metrics/Multiplayer/Active=true` value? This doesn't interact with the AWS metrics gem, it enables the native O3DE metrics logging in the Multiplayer gem.

the previous setting, `/Amazon/AWSCore/AllowAWSMetadataCredentials=true`, _is_ required for the metrics gem to work, however it's also required for _any_ AWS integrations to work. And since we're running on EC2 felt like a good thing to have enabled by default

Reply:
Thanks for the reply. I misread the set reg path and see that its the core metrics and not the optional aws metrics setting.